2-overview-functions.md

```diff
@@ -1,14 +1,12 @@
-# Functions
-
-At its core, a function is a block of code we can execute on demand. This block of code might perform a task, such as navigate to a different part of a website, or calculate a value, like the sum of two numbers. Functions are also perfect for scenarios where we need to perform the same task multiple times; rather than duplicating the logic in multiple locations (which would make it hard to update when the time comes), we can centralize it in one location, and call it whenever we need the operation performed - you can even call functions from other functions!
+Functions are a key building block for writing code in JavaScript. A function is a reusable collection of lines of code that makes something happen within your program. Functions are perfect for scenarios where we need to perform the same task multiple times; rather than duplicating the logic in multiple locations (which would make it hard to update when the time comes), we can centralize it in one location, and call it whenever we need the operation performed - you can even call functions from other functions!
```

Comment:
```suggestion
Functions are a key building block for writing code. A function is a reusable collection of lines of code that makes something happen within your program. Functions are perfect for scenarios where we need to perform the same task multiple times; rather than duplicating the logic in multiple locations (which would make it hard to update when the time comes), we can centralize it in one location, and call it whenever we need the operation performed - you can even call functions from other functions!
```